lib/doro/behaviors/god.ex

```diff
@@ -20,4 +21,19 @@ defmodule Doro.Behaviors.God do
       send_to_player(player, "```\n#{Poison.encode!(chunk, pretty: true)}\n```")
     end)
   end
+
+  interact_if("/edit", %{player: player, rest: rest}) do
+    found_entity = Doro.World.get_entities([in_location(player[:location]), named(rest)])
+    |> Enum.at(0)
+    rest_present = (rest != nil) && (String.length(rest |> String.trim()) > 0)
+    rest_present && found_entity
+  end
+
+  interact("/edit", %{player: player, rest: name}) do
+    entity = Doro.World.get_entities([in_location(player[:location]), named(name)]) |> Enum.at(0)
+    send_to_player(player,
+      "You concentrate on #{definite(entity)}.  Nothing happens and you feel silly.",
+      entity
```

Comment:
Before this is supported in the HTML front end, this needs to be augmented with a `type`, or something like that, right?  Just wondering. 

```json
{
  "text": "You concentrate on the entity, etc.",
  "data": {
    "type": "open_entity_editor",
    "entity": {"id": "turntable" }
  }
}
```

```json
{
  "text": "You hear a sound.",
  "data": {
    "type": "play_sound",
    "url": "http://environmentalsounds.com/creepy_forest.wav"
  }
}
```